### PR TITLE
[ Bug fixed ] : Popover content in left sidebar is not scrollable

### PR DESCRIPTION
**Pull request by manishkushare**
Resolves 
Screenshot 2023-01-04 at 10 01 12 PM

Screenshot 2023-01-04 at 10 01 25 PM

Screenshot 2023-01-04 at 10 01 42 PM

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-85zrhe1va Popover content is not scrollable in left sidebar](https://app.clickup.com/t/85zrhe1va)

File: frontend/src/_ui/Popover/index.jsx [hunk 1]
```diff
@@ -14,16 +14,30 @@ const PopoverComponent = ({
...
-
+  const computeStyle = () => {
+    if (popoverContentHeight) {
+      return {
+        height: `${popoverContentHeight}vh`,
+        overflow: 'auto',
+      };
+    } else {
+      return {};
+    }
```
Comment (kavinvenkatachalam) [inline frontend/src/_ui/Popover/index.jsx:28]:
```suggestion
    } 
      return {}
```

No need else statement as we're returning in if

File: frontend/src/Editor/LeftSidebar/index.jsx [hunk 2]
```diff
@@ -46,10 +46,15 @@ export const LeftSidebar = forwardRef((props, ref) => {
...
     dataQueries,
     clonePage,
+    queryPanelHeight,
   } = props;
   const [selectedSidebarItem, setSelectedSidebarItem] = useState();
   const [showLeaveDialog, setShowLeaveDialog] = useState(false);
   const [showDataSourceManagerModal, toggleDataSourceManagerModal] = useState(false);
+  const [popoverContentHeight, setPopoverContentHeight] = useState(queryPanelHeight);
+  useEffect(() => {
+    setPopoverContentHeight(queryPanelHeight);
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/LeftSidebar/index.jsx:56]:
```suggestion
    popoverContentHeight !== queryPanelHeight && setPopoverContentHeight(queryPanelHeight);
```

Update the height only if both are not equal. It avoids unwanted renders

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@manishkushare Requested small changes

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**